docs/operate-and-deploy/changelog.md

```diff
@@ -9,6 +9,7 @@ keywords: ksqldb, changelog
 Version 0.12.0
 --------------
 
+- [ksqlDB 0.12.0 Introduces Real-Time Query Upgrades and Automatic Query Restarts](https://www.confluent.io/blog/ksqldb-0-12-0-features-updates/)
 - [ksqlDB v0.12.0 changelog](https://github.com/confluentinc/ksql/blob/master/CHANGELOG.md#0120-2020-08-26)
 
 Version 0.11.0
```

Comment:
Any reason we don't link to the 0.11.0 blog post?

Reply:
@vcrfxia Well spotted! Looks like I missed a cherry-pick somewhere along the line. Maybe someday we'll get pint merge for these branches. A guy can dream. https://github.com/confluentinc/ksql/blob/0.11.0-ksqldb/docs/operate-and-deploy/changelog.md